scripts/install.sh

```
@@ -0,0 +1,158 @@
+cp -R scripts "${STAGING}/scripts"
+
+# node_modules / cdk.out / dist を完全に排除 (CodeBuild の npm install が EEXIST で失敗するため)
+find "${STAGING}" -type d \( -name node_modules -o -name cdk.out -o -name dist \) -prune -exec rm -rf {} +
+find "${STAGING}" -name ".DS_Store" -delete
+
+# TenkaCloud ルートの絶対パスを記憶して最後に server/ に戻る
+TENKACLOUD_ROOT="$(pwd)"
+
+cd "${STAGING}"
+zip -rq "${CDK_SOURCE_NAME}" .
+export CDK_PARAM_COMMIT_ID=$(aws s3api put-object --bucket "${CDK_PARAM_S3_BUCKET_NAME}" --key "source.zip" --body "./${CDK_SOURCE_NAME}" --output text)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)